packages/expo-updates/ios/EXUpdates/Database/EXUpdatesDatabase.h

```diff
@@ -33,6 +33,13 @@ typedef NS_ENUM(NSInteger, EXUpdatesDatabaseHashType) {
 - (void)deleteUpdates:(NSArray<EXUpdatesUpdate *> *)updates error:(NSError ** _Nullable)error;
 - (nullable NSArray<EXUpdatesAsset *> *)deleteUnusedAssetsWithError:(NSError ** _Nullable)error;
 
+/**
+ * This method ignores the scopeKey field in the config object and returns ALL updates in the database.
+ * The config object is passed along to each update object, even if its scopeKey doesn't match.
+ *
+ * This method should only be used to introspect the state of the database, and the update objects
+ * returned should not be used for any other purpose.
```

Comment:
I'm a bit confused by this comment and the restriction it is imposing. The method is used here to generate the list of updates to reap:

https://github.com/expo/expo/blob/master/packages/expo-updates/ios/EXUpdates/Database/EXUpdatesReaper.m#L26

What is an example of an activity that you would want prohibited?

Reply:
Good point; I mainly wanted to indicate that this method shouldn't be used to select updates that will then be used to launch the app. Tried to clarify the comment; let me know if you think this makes more sense.